code/core/trkCore.cc

```diff
@@ -1288,7 +1288,7 @@ float addInputsToLineSegmentTracking(SDL::Event &event, bool useOMP)
         float py = p3LH.Y();
         float pz = p3LH.Z();
 
-        if ((ptIn > 0.7) and (fabs(p3LH.Eta()) < 3))
+        if ((ptIn > 0.99) and (fabs(p3LH.Eta()) < 3))
```

Comment:
I thought that there was supposed to be `1.0 - 2*ptErr`

Reply:
next PR will have it (or I'll add that commit to this PR here later today)